starter-code/routes/drones.js

```diff
@@ -1,21 +1,56 @@
 const express = require('express');
 
 // require the Drone model here
+const Drone = require('../models/drone');
 
 const router = express.Router();
 
 
-router.get('/drones', (req, res, next) => {
+router.get('/', (req, res, next) => {
   // Iteration #2
+  Drone.find({}, (err, result) => {
+    if(err){
+      next();
+    }
+    else{
+      let drones = {
+        droneName : result.name,
+        droneProp : result.propeller,
+        droneMaxSpeed : result.maxSpeed,
+        drones : result
+      }
+
+      res.render('drones/index', drones)
```

Comment:
What are you doing here? result is not a single drone, is an array of drones. You want to pass all of them to the render:
```
res.render('drones/index', {drones: result});
```
Of course, you don't need the code from line 16 to line 21.